app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    """ That will move a file from one location to another """
+    # Ex: mv file.txt first_dir/second_dir/third_dir/file2.txt
+
+    (cmd,
+     file_in,
+     file_out) = (
+        command.split(" ")[0],
+        command.split(" ")[1],
+        command.split(" ")[2])
+
+    if cmd != "mv":
+        raise ValueError("Command mv not found")
+
+    if "/" in file_out:
+        dir_path = ""
+
+        for directory in os.path.dirname(file_out).split("/"):
```

Comment:
Use `os.path.split` instead of `split("/")`

Reply:
this will give me a string, but I need to split it into / to create each directory

Reply:
os.path.split returns tuple of 2 elements, where first element is directories. It's enought to create directories